index.html

```diff
@@ -85,13 +84,79 @@
 			</ul>
 		</div>
 		<div class="product-details-container">
+			<div class="product-redirect">Back to <a href="#" class="product-selected-category">Woman<a/></div>
+			<h1 class="product-name">The Atelier Tailored Coat</h1>
+			<div class="rating-wrapper">
+			</div>
+			<div class="product-rating-review">
+				<div class="product-counter-review"><!-- it is a dynamic block to display reviews level -->
+				</div>
+				Review(s) <a href="#" class="product-review-button">add review</a>
+			</div>
+			<div class="product-price">&#36;499.00</div>
+			<ul class="product-additional-info">
+				<li class="product-additional-info-availability"><span class="product-additional-info-property">Availability :</span><span class="product-additional-info-value">In stock</span></li>
+				<li class="product-additional-info-availability">Product code :<span class="product-additional-info-value"> &#35;4657</li>
+				<li class="product-additional-info-availability">Tags:
+					<a href="#" class="product-fashion">Fashion</a>
+					<a href="#" class="product-hood">Hood</a>
+					<a href="#" class="product-classic">Classic</a>
```

Comment:
you don't need specific class here - you just need one generic class... e.g., "product-tag"